tinygrad/runtime/ops_kfd.py

```diff
@@ -363,10 +371,12 @@ def _wait_signal(self, signal:hsa.amd_signal_t, timeout=60000):
   def __init__(self, device:str=""):
     if KFDDevice.kfd == -1: KFDDevice.kfd = os.open("/dev/kfd", os.O_RDWR)
     self.device_id = int(device.split(":")[1]) if ":" in device else 0
-    with open(node_sysfs_path(self.device_id+1, "gpu_id"), "r") as f: self.gpu_id = int(f.read())
-    with open(node_sysfs_path(self.device_id+1, "properties"), "r") as f: self.properties = {line.split()[0]: int(line.split()[1]) for line in f}
+    first_gpu = find_first_gpu_node()
+    with open(node_sysfs_path(self.device_id+first_gpu, "gpu_id"), "r") as f: self.gpu_id = int(f.read())
+    with open(node_sysfs_path(self.device_id+first_gpu, "properties"), "r") as f: self.properties = {line.split()[0]: int(line.split()[1]) for line in f}
     self.drm_fd = os.open(f"/dev/dri/renderD{self.properties['drm_render_minor']}", os.O_RDWR)
-    self.arch = f"gfx{self.properties['gfx_target_version']//100}"
+    target = int(self.properties['gfx_target_version'])
+    self.arch = "gfx%d%x%x" % (target // 10000, (target // 100) % 100, target % 100)
```

Comment:
Should we change that? We get major and minor which are passed to the compiler (like `gfx1100`), why should we care about revision?